src/components/section-features.tsx

```diff
@@ -0,0 +1,71 @@
+import { featuresSectionData } from '@/lib/constants';
+import { getImageUrl } from '@/lib/utils';
+import Image from 'next/image';
+
+interface ProductFeatureCardProps {
+  title: string;
+  description: string;
+  imageUrl: string;
+  reverse?: boolean;
+}
+
+export function ProductFeatureCard({
+  title,
+  description,
+  imageUrl,
+  reverse = false,
+}: ProductFeatureCardProps) {
+  return (
+    <div
+      className={`flex flex-col md:flex-row items-center justify-between gap-8 py-6 ${
+        reverse ? 'md:flex-row-reverse' : ''
+      }`}
+    >
+      <div className="w-full md:w-1/2 space-y-4">
+        <h3 className="lg:text-4xl text-2xl text-blue-950 font-bold">
+          {title}
+        </h3>
+        <p className="text-gray-700 text-lg lg:text-xl">{description}</p>
+      </div>
+      <div className="w-full md:w-1/2">
+        <Image
+          src={getImageUrl(imageUrl)}
+          alt={title}
+          width={300}
+          height={225}
+          className="object-cover rounded-lg w-full"
+        />
+      </div>
+    </div>
+  );
+}
+
+export function FeaturesSection() {
+  const { sectionId, title, subtitle, features } = featuresSectionData;
+
+  return (
+    <section id={sectionId} className="py-20 bg-blue-200">
+      <div className="container mx-auto max-w-6xl px-6">
```

Comment:
```suggestion
      <div className="container mx-auto px-6">
```
That's what's preventing features from occupying the whole container